docs/data-management/delete.md

```diff
@@ -96,15 +96,18 @@ The available grammar is:
     "dataSource": <task_datasource>,
     "interval" : <all_unused_segments_in_this_interval_will_die!>,
     "context": <task context>,
-    "batchSize": <optional_batch size>
+    "batchSize": <optional_batch size>,
+    "maxSegmentsToKill": <the maximum number of segments to delete>
 }
 ```
 
 Some of the parameters used in the task payload are further explained below:
 
-| Parameter    |Default| Explanation                                                                                            |
-|--------------|-------|--------------------------------------------------------------------------------------------------------|
+| Parameter   | Default         | Explanation                                                                                                                                                                                                                                                                                                                                                                   |
+|-------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `batchSize`    |100    | Maximum number of segments that are deleted in one kill batch. Some operations on the Overlord may get stuck while a `kill` task is in progress due to concurrency constraints (such as in `TaskLockbox`). Thus, a `kill` task splits the list of unused segments to be deleted into smaller batches to yield the Overlord resources intermittently to other task operations.|
+| `limit`     | null - no limit | Maximum number of segments that are deleted in the kill task. This property may not be combined with `markAsUnused: true`                                                                                                                                                                                                                                                     |
```

Comment:
is the key called `limit` or `maxSegmentsToKill` (line 100)?

Reply:
Thanks! Fixed